File: Average.java
//to find the avaerage of n numbers using array
import java.util.*;
public class Average {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double avg=0.0;
        int sum=0;
        System.out.println("Enter the size of the array");
        int n = sc.nextInt();
        int[] arr = new int[n];
        System.out.println("Enter the elements of the array");
        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
            sum=sum+arr[i];
        }
        System.out.println();
        System.out.println("The Given array is :");
        for (int i = 0; i < n; i++) {
            System.out.print(arr[i] + " ");
        }
        System.out.println();
        System.out.println("The Sum of Array is :"+ sum);
        avg=sum/n;
        System.out.println("Average is :"+ avg);
    }
}

File: average.c
// Average Of Array;
#include<stdio.h>
int main()
{
    int n,sum=0;
    float avg;
    printf("Enter The Size Of Array :\n");
    scanf("%d", &n);
    int a[n];
    printf("Enter Array Elements :");
    for (int i = 0; i < n; i++)
    {
        scanf("%d", &a[i]);
        sum=sum+a[i];
    }
    avg=sum/n;
    printf("Given Array is :\n");
    for (int i = 0; i < n; i++)
    {
        printf("%d ", a[i]);
    }
    printf("\n");
    printf("Sum Of the Given Array is :%d", sum );
    printf("\n");
    printf("Average Of the Given Array is :%f", avg);
    return 0;
}